css/main.css

```diff
@@ -576,7 +650,642 @@ footer {
+        flex-direction: column;
+        text-align: center;
+    }
+
+    .jobs-screen__title {
+        margin-right: 0;
+    }
+
+    .jobs-screen__link {
+        margin-top: 50px;
+        margin-left: 0;
+    }
+
+    .jobs-list {
+        flex-direction: column;
+    }
+
+    .job:nth-of-type(2) {
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    .our-projects__examples {
+        flex-wrap: wrap;
+    }
+
+    .our-projects__example {
+        flex: 0;
+        flex-basis: 44%;
+    }
+
+    .our-projects__example-1 {
+        margin-left: 15px;
+    }
+
+    .our-projects__example-4 {
+        margin-right: 15px;
+    }
+
+    .our-projects__example-3, .our-projects__example-4 {
+        margin-top: 15px;
+    }
+
+    .video-screen__text-block {
+        padding-bottom: 310px;
+    }
+
+    .video-wrapper {
+        margin-top: -240px;
+    }
+
+    .our-projects__text {
+        padding: 50px 90px 70px;
+    }
+}
+
+@media (max-width: 539px) {
+    .our-projects__examples {
+        flex-direction: column;
+    }
+
+    .our-projects__example-2 {
+        margin-top: 15px;
+    }
+}
+
+@media (max-width: 480px) {
+    * {
+        font-size: 14px;
+    }
+
+    h1 {
+        font-size: 36px;
+    }
+
+    h2 {
+        font-size: 26px;
+        line-height: 40px;
+    }
+
+    h3 {
+        font-size: 16px;
+    }
+
+    .welcome-screen__slogan-block {
+        margin: 20px 10px 50px;
+    }
+
+    .examples-screen__title {
+        margin-left: 10px;
+        margin-right: 10px;
+    }
+
+    .examples-screen__content {
+        margin: 50px 30px 70px;
+    }
+
+    .our-projects-screen {
+        padding-top: 70px;
+    }
+
+    .our-projects__example {
+        flex: 0;
+        flex-basis: 22%;
+    }
+
+    .advantage {
+        margin: 0 0 40px 0;
+    }
+
+    .high-dev-standards {
+        padding-top: 50px;
+    }
+
+    .email-screen__input-block {
+        padding: 0 20px;
+    }
+
+    .email-screen__input {
+        padding-right: 0;
+        padding-left: 10px;
+    }
+
+    .email-screen__input-button {
+        padding: 15px;
+    }
+
+    .our-projects__text {
+        padding: 50px 30px 70px;
+    }
+
+    .video-screen__text-block {
+        padding-bottom: 220px;
+    }
+
+    .video-wrapper {
+        margin-top: -150px;
+    }
+
+    .video-block-border {
+        margin: 0;
+    }
+
+    footer {
+        padding-top: 0;
+    }
+
+    .footer-wrapper {
+        height: 0;
+    }
+
+    .footer-main-page {
+        display: none;
+    }
+
+    .form__inner {
+        padding: 40px 10px;
+    }
+
+    .feedback-form__input {
+        margin: 10px 15px;
+    }
+
+    .feedback-form__input_phone-number {
+        margin-bottom: 30px;
+    }
+}
+
+@media (max-width: 380px) {
+    .email-screen__input {
+        width: 170px;
+    }
+
+    .form {
+        width: 300px;
+    }
+
+    .form__send-button {
+        padding: 15px 30px;
+    }
+}
+
+@media (max-width: 320px) {
```

Comment:
Just add `min-width: 320px;` for `html, body` at the start oft the file. And remove `@media (max-width: 320px)` fully.